Src/WitsmlExplorer.Frontend/services/uidMappingService.tsx

```
@@ -36,6 +40,21 @@ export default class UidMappingService {
     }
   }
 
+  public static async getUidMappingBasicInfos(
+    abortSignal?: AbortSignal
+  ): Promise<UidMappingBasicInfo[]> {
+    const response = await ApiClient.get(
+      `/api/uidmapping/basicinfos`,
+      abortSignal,
+      undefined
+    );
+    if (response.ok) {
+      return response.json();
+    } else {
+      return [];
```

is that wanted behavior to hide potential errors?

---

The same solution is elsewhere. Error is still being logged, but it doesn't bother user with unnecessary information. Anyway this error should mostly happen when connection to server is lost.